rpgJavaInterpreter-core/src/test/kotlin/com/smeup/rpgparser/utils/DbMock.kt

```diff
@@ -44,10 +45,19 @@ interface DbMock : AutoCloseable {
         )
     }
 
-    fun <R> usePopulated(predicate: (DbMock) -> R) = this.use {
-        val queries = listOf(it.createTable(), it.populateTable())
+    fun <R> usePopulated(
```

Comment:
Could not you implement only one `usePopulated` function with the `values`  initialized to `emptyList()`?